.github/workflows/release.yml

```diff
@@ -57,7 +65,24 @@ jobs:
           rustup component add clippy --toolchain stable-x86_64-apple-darwin
           rustup update stable
 
+      - name: setup for cross-compile builds
+        if: runner.os == 'linux'
```

Comment:
We should make this `if` statement use something in the matrix config so we don't end up compiling OpenSSL during a normal linux build

Reply:
hmm... good idea. perhaps `matrix.config.arch == "aarch64"` would do